components/camel-dynamic-router/pom.xml

```diff
@@ -69,4 +69,23 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
```

Comment:
Is these settings really needed, we dont have this kind in the other compomnents

Reply:
I am really not sure what I was doing there, so I will just remove it.